environment.yml

```diff
@@ -0,0 +1,14 @@
+name: classroom_abm
+channels:
+  - anaconda
+  - conda-forge
+  - defaults
+dependencies:
+  - black
+  - conda-lock
+  - pre_commit
+  - mesa
+  - numpy
+  - pandas
+  - scipy
+prefix: /opt/homebrew/Caskroom/miniconda/base/envs/mesa_env
```

Comment:
I think we can remove the `prefix` line as it maps to a path that's specific to your machine.